### PR TITLE
translations: init danish

### PR DESCRIPTION
**Pull request by AngryAnt**
First pass. Content complete, but I need to do another pass and a final typo check.

### CODE REVIEW

File: src/pages/dk/index.astro [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+			<div class="prose prose-invert py-16 px-4 max-w-3xl">
+				<Values />
+				<Goals />
+				<Roadmap />
+			</div>
+		</div>
+	</main>
+</Layout>
+
+<style></style>
```
Comment (mrshmllow) [inline src/pages/dk/index.astro]:
```suggestion
---
import Layout from "../../layouts/Layout.astro";
import Home from "../../components/home/Home.astro";
---

<Layout title="auxolotl.org">
	<Home />
</Layout>```

**Review by mrshmllow — CHANGES_REQUESTED** [review]

File: src/i18n/ui.ts [hunk 2]
```diff
@@ -1,11 +1,14 @@
 import en from "../../public/locales/en/translations.json"
+import dk from "../../public/locales/dk/translations.json"
```
Comment (dantefromhell) [inline src/i18n/ui.ts]: nitpick: I vote for sorting alphabetically - IMO makes finding things easier.

**Review by mrshmllow — APPROVED** [review]

**Comment by AngryAnt** [comment]
Final pass done, typo check done, rebased on main.

Not sure who can merge - beyond tossing it at an auto-translator to verify I didn't just upload my manifest on the superiority of pygmy owls?

**Comment by mrshmllow** [comment]
If you believe its done, I am happy to merge!

**Pull request merged**